File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int expectedsum=(n*(n+1))/2;
        int actualsum=0;
        for(int i: nums){
            actualsum +=i;
        }
        int missingno =expectedsum - actualsum;
        return missingno;
    }
}

File: 0268-missing-number/0268-missing-number.cs
public class Solution {
    public int MissingNumber(int[] nums) {
        int n=nums.Length;
        int expectedSum=(n*(n+1))/2;
        int actualSum=0;
        foreach(int val in nums){
            actualSum+=val;
        }
        int missinNo=expectedSum-actualSum;
        return missinNo;
    }
}